### PR TITLE
Go version update in go.mod and fsinfo used from gofsutils

### PR DESCRIPTION
**Pull request by Sakshi-dell**
# Description
1. Updated go version to 1.17 in go.mod
2. used FsInfo from gofsutils instead of k8s

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/171 |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: common/k8sutils/k8sutils.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func LeaderElection(clientset *kubernetes.Clientset, lockName string, namespace
 
 //GetStats - Returns the stats for the volume mounted on given volume path
 func GetStats(ctx context.Context, volumePath string) (int64, int64, int64, int64, int64, int64, error) {
-	availableBytes, totalBytes, usedBytes, totalInodes, freeInodes, usedInodes, err := fs.Info(volumePath)
+	availableBytes, totalBytes, usedBytes, totalInodes, freeInodes, usedInodes, err := gofsutil.FsInfo(ctx, volumePath)
```
Comment (bpjain2004) [inline common/k8sutils/k8sutils.go:67]: remove this empty line

Comment (Sakshi-dell) [reply common/k8sutils/k8sutils.go]: updated

**Review by randeepsharma — APPROVED** [review]
LGTM

File: common/k8sutils/k8sutils.go [hunk 2]
```diff
@@ -3,13 +3,13 @@ package k8sutils
 import (
 	"context"
 	"fmt"
+	"github.com/dell/gofsutil"
```
Comment (prablr79) [inline common/k8sutils/k8sutils.go:6]: hope this has been tested and found no issues.

**Review by prablr79 — APPROVED** [review]
conditional approval, hope all dependent lib is validated

File: common/k8sutils/k8sutils.go (hunk 2 above)
Comment (Sakshi-dell) [reply common/k8sutils/k8sutils.go:6]: It has been tested and no issues were found.

**Pull request merged**